### PR TITLE
ICS 7: Tendermint Client

### PR DESCRIPTION
**Pull request by cwgoes**
Ref https://github.com/cosmos/ics/issues/286
Closes https://github.com/cosmos/ics/issues/311
Closes https://github.com/cosmos/ics/issues/145
Closes https://github.com/cosmos/ics/issues/13
Ref https://github.com/cosmos/ics/issues/282; that will be a separate spec though.

- [x] Alter paths for non-conflicting human-readable prefixes, e.g. `/consensusState/{client-identifier}`

### CODE REVIEW

**Comment by cwgoes** [comment]
Not yet totally complete but ready for initial review.

File: spec/ics-007-tendermint-client/README.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+  height: uint64
+  commitmentRoot: []byte
+  validatorSet: List<Pair<Address, uint64>>
+  signatures: []Signature
+}
+```
+
+### Evidence
+
+Tendermint client `Evidence` consists of two headers at the same height both of which the light client would have considered valid.
```
Comment (fedekunze) [inline spec/ics-007-tendermint-client/README.md]: Explain that the `Evidence` type is used for punishing client misbehaviour and freezing it if applicable.

File: spec/ics-007-tendermint-client/README.md [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+}
+```
+
+### Client initialisation
+
+Tendermint client initialisation requires a (subjectively chosen) latest consensus state, including the full validator set.
+
+```typescript
+function initialize(consensusState: ConsensusState, validatorSet: List<Pair<Address, uint64>>, latestHeight: uint64): ClientState {
+  return {
```
Comment (fedekunze) [inline spec/ics-007-tendermint-client/README.md]:
```suggestion
  return ClientState{
```

File: spec/ics-007-tendermint-client/README.md (hunk 1 above)
Comment (cwgoes) [reply spec/ics-007-tendermint-client/README.md]: Added.

File: spec/ics-007-tendermint-client/README.md (hunk 2 above)
Comment (cwgoes) [reply spec/ics-007-tendermint-client/README.md]: Updated.

**Comment by cwgoes** [comment]
Merging, but more reviews are welcome anytime.

**Pull request merged**